### ISSUE TITLE
/ci command needs a permission node

### ISSUE THREAD

**Issue by xXRyzen** [Shelly7w7/ShellyEssentials-#12]
Noticed players screaming "Who cleared my inventory", when i checked console i noticed someone trolling using /ci <name>. Should be an op only command. Thanks in advanced!

**Comment by iiFlamiinBlaze** [comment]
It already is an OP only cmd (I checked the /clearinv command code and plugin.yml), so it isn't an issue with the base form of this plugin. Did you edit any of the plugin.yml? Also do you have the most updated version of ShellyEssentials?

**Comment by xXRyzen** [comment]
Yes i have a latest version of the plugin, and yes /clearinv is an op command, but /ci is acting like a short version of /clearinv that can be used by non ops

**Comment by Shelly7w7** [comment]
> Yes i have a latest version of the plugin, and yes /clearinv is an op command, but /ci is acting like a short version of /clearinv that can be used by non ops

Are you sure you are not using any third party essentials plugin which is overwriting mine /ci command?
